Fix ``TypeError`` for ``HelpFormatter`` with Python 3.14
